### PR TITLE
Language: Missing keys for pt and pt-BR

### PR DESCRIPTION
**Pull request by camponez**

### CODE REVIEW

**Review by mjankowski — CHANGES_REQUESTED** [review]
It looks like there are parse errors with the yaml here, can you investigate?

**Comment by camponez** [comment]
Ok, I look at all keys. I don't get what
` can not load translations from /home/travis/build/tootsuite/mastodon/config/locales/pt-BR.yml: #<Psych::SyntaxError: (/home/travis/build/tootsuite/mastodon/config/locales/pt-BR.yml): did not find expected key while parsing a block mapping at line 309 column 5>`
mean exactly.

File: config/locales/pt-BR.yml [hunk 1]
```diff
@@ -198,6 +274,51 @@ pt-BR:
...
+    formats:
+      default: "%d %b, %Y, %H:%M"
+  two_factor_authentication:
+    code_hint: Entre o código gerado pelo seu aplicativo para confirmar
+    description_html: Se você habilitar <strong>autenticação two-factor </strong>, quando logar será necessário o seu telefone que vai gerar os tokens usado.
+    disable: Disabilitar
+    enable: Habilitar
+    enabled_success: Autenticador Two-factor habilitador com sucesso
+    generate_recovery_codes: Gerar códigos para recuperar conta
+    instructions_html: "<strong>Scaneie este código QR no seu Google Authenticator ou aplicativo similar no seu telefone</strong>. A partir de agora seu aplicativo irá gerar tokens que deverão ser digitados para você logar".
```
Comment (nightpool) [inline config/locales/pt-BR.yml]: the period here has to be inside of the quotation marks, or else it's a syntax error. the quotation marks aren't part of the string, they're just part of the yml literal syntax

File: config/locales/pt.yml [hunk 2]
```diff
@@ -202,6 +213,45 @@ pt:
...
+    formats:
+      default: "%b %d, %Y, %H:%M"
+  two_factor_authentication:
+    code_hint: Entre o código gerado pelo seu aplicativo para confirmar
+    description_html: Se você habilitar <strong>autenticação two-factor </strong>, quando logar será necessário o seu telefone que vai gerar os tokens usado.
+    disable: Disabilitar
+    enable: Habilitar
+    enabled_success: Autenticador Two-factor habilitador com sucesso
+    generate_recovery_codes: Gerar códigos para recuperar conta
+    instructions_html: "<strong>Scaneie este código QR no seu Google Authenticator ou aplicativo similar no seu telefone</strong>. A partir de agora seu aplicativo irá gerar tokens que deverão ser digitados para você logar".
```
Comment (nightpool) [inline config/locales/pt.yml]: same as above

File: config/locales/pt-BR.yml (hunk 1 above)
Comment (camponez) [inline config/locales/pt-BR.yml]: Thanks. Missed that!

**Comment by nightpool** [comment]
I recommend using http://yaml-online-parser.appspot.com/ to test your YML if you have issues with it. it's error messages are often more informative then the standard ruby ones.

**Review by mjankowski — APPROVED** [review]

**Pull request merged**